os/android.toolchain.cmake

```diff
@@ -338,11 +341,11 @@ endmacro()
 
 
 # fight against cygwin
-set( ANDROID_FORBID_CYGWIN TRUE CACHE BOOL "Prevent cmake from working under cygwin and using cygwin tools")
```

Comment:
This one is an error actually. I've send pull request last year but it still not merged (and not rejected): https://github.com/taka-no-me/android-cmake/pull/50